FunctionalPreprocessing/FunctionalPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -167,17 +220,43 @@ main()
 
 	get_options "$@"
 
-	local directory=${XNAT_PBS_JOBS_RUNNING_STATUS_DIR}/${g_project}
-	local file="FunctionalPreprocessing.${g_subject}_${g_classifier}_${g_scan}.RUNNING"
+	local directory="${XNAT_PBS_JOBS_BUILD_DIR}/${g_project}/FunctionalPreprocessing.${g_session}_${g_scan}_RUNNING_STATUS"
```

Comment:
Since you are using `XNAT_PBS_JOBS_BUILD_DIR` in the script, you should treat it like `XNAT_PBS_JOBS` is treated at the top of the script.

That is, you should put statements at the top of the script that check to make sure that `XNAT_PBS_JOBS_BUILD_DIR` actually has a value just like we make sure the `XNAT_PBS_JOBS` actually has a value.

The script should immmediately report an error and abort if the `XNAT_PBS_JOBS_BUILD_DIR` environment variable is not set.

Same thing holds for the `XNAT_PBS_JOBS_ARCHIVE_ROOT` variable.

Reply:
added